prometheus_hardware_exporter/collector.py

```diff
@@ -845,4 +916,5 @@ def process(self, payloads: List[Payload], datastore: Dict[str, Payload]) -> Lis
     "lsi-sas-3-collector": LSISASControllerCollector(3),
     "poweredge-raid-collector": PowerEdgeRAIDCollector(),
     "hpe-ssa-collector": SsaCLICollector(),
+    "redfish-collector": RedfishCollector("", "", ""),
```

Comment:
**issue (non-blocking):** 

I guess here somehow bring out an issue: what if the collector need some config to initialize?

I will suggest to refactor this part in later pr.

Reply:
This is now removed.